mysql-test/t/slow_log_engine_fields.test

```diff
@@ -0,0 +1,16 @@
+SET SESSION long_query_time = 0;
+
+CREATE TABLE t1 (A INT);
+INSERT INTO t1 VALUES (1),(2),(3);
+
+--let SEARCH_FILE = $MYSQLTEST_VARDIR/mysqld.1/mysqld-slow.log
+
+# Check if Semisync_ack_time is included in slow log results
+--let SEARCH_PATTERN= Semisync_ack_time:
+--source include/search_pattern_in_file.inc
```

Comment:
Can there be a pre-existing slow log, resulting in false positives?

Reply:
This feature is compiled inside a server so all comments in slow log should have "Semisync_ack_time" or none. For this case "--repeat" doesn't matter.